### PR TITLE
[BPK-1068] fix iOS Podspec

### PR DESCRIPTION
**Pull request by matteo-hertel**

### CODE REVIEW

**Comment by georgegillams** [comment]
Sweet!

File: native/packages/react-native-bpk-component-icon/BpkIcon.podspec [hunk 1]
```diff
@@ -8,6 +8,7 @@ Pod::Spec.new do |s|
   s.homepage       = "https://github.com/Skyscanner/backpack/tree/master/packages/react-native-bpk-component-icon"
   s.license        = "Apache License 2.0"
   s.author         = { "Backpack" => "backpack@skyscanner.net" }
+  s.source         = { :path => './' }
   s.resources      = "./node_modules/bpk-svgs/dist/font/*.ttf"
```
Comment (matthewdavidson) [inline native/packages/react-native-bpk-component-icon/BpkIcon.podspec]: Although now i am looking at this:
```
s.resources      = "./node_modules/bpk-svgs/dist/font/*.ttf"
```

And wondering if it's correct 🤔  

I guess it works for you?

Comment (matteo-hertel) [reply native/packages/react-native-bpk-component-icon/BpkIcon.podspec]: yes, it's installing everything properly without complaining 🤞

Comment (k0nserv) [reply native/packages/react-native-bpk-component-icon/BpkIcon.podspec]: Yeah this seems problematic since it relies on the resolution/deduping by NPM

Comment (matthewdavidson) [reply native/packages/react-native-bpk-component-icon/BpkIcon.podspec]:
```
node_modules/
  react-native-bpk-component-icon/
    BpkIcon.podspec
  bpk-svgs/
```
^ thats how the dir structure should look after `npm install`

but that `BpkIcon.podspec` points to a relative path (`./node_modules/bpk-svgs/dist/font/*.ttf`):
 ```
node_modules/
  react-native-bpk-component-icon/
    node_modules/
      bpk-svgs/                  // < ---- almost like it's looking here
    BpkIcon.podspec
  bpk-svgs/
```

**Pull request closed without merging**